tdrs-frontend/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
```

Comment:
I know this was copied over from trussworks, but since we moving it in are you considering moving the project over to `.tsx`? if not it feels a bit inconsistent to have one component as tsx.

Reply:
@lauraGgit Sorry, I just had the tsx as a middle step. It is moved over to jsx and I removed the deprecated lines as well. That should have been pushed last night so let me know if you're not seeing it now.